README.md

```diff
@@ -35,11 +38,32 @@ The view options can be set in 3 different methods:
 
 ---
 
+## Feed Integration example app
+
+This example app demonstrates the following scenarios of `FwVideoFeedView` integration: The app
+shows how to integrate
+`FwVideoFeedView` in activity, fragment, recycler view, and remove/add `FwVideoFeedView` from the
+container on the runtime.
+
+* `FwVideoFeedView` integrated in activity
+* `FwVideoFeedView` integrated in fragment
+* `FwVideoFeedView` integrated in recycler view
+* `FwVideoFeedView` add/removed from the container on the runtime.
+
+[Feed Integration example app](feed_integration)
+
+| FwVideoFeedView in Activity                                        | FwVideoFeedView in recycler view                                   |
+|--------------------------------------------------------------------|--------------------------------------------------------------------|
+| ![Feed Integration example app](feed_integration/Screenshot1.jpeg) | ![Feed Integration example app](feed_integration/Screenshot2.jpeg) |
```

Comment:
I believe we can add more screenshots from all screens, including the selector screen, and 4 different integrations.